src/auspex/experiment.py

```diff
@@ -513,6 +513,14 @@ def run_sweeps(self):
             for w in wrs:
                 w.filename.value = inc_filename
         self.filenames = [w.filename.value for w in self.writers]
+        # Save ChannelLibrary version
```

Comment:
I don't think I agree with this approach, at least not unilaterally. User should have the option of not applying changes, for example. If somebody runs an experiment in a loop this is going to create an enormous database.

Reply:
Fair enough, I agree it should be optional. This was already merged to develop, I'll change it from there